hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -430,14 +430,14 @@ private FlinkOptions() {
   public static final ConfigOption<Boolean> COMPACTION_SCHEDULE_ENABLED = ConfigOptions
       .key("compaction.schedule.enabled")
       .booleanType()
-      .defaultValue(true) // default true for MOR write
-      .withDescription("Schedule the compaction plan, enabled by default for MOR");
+      .defaultValue(false) // default false for MOR write
```

Comment:
@danny0405 WDYT of changing the default value? We could move all the compaction and cleaning stuff to the batch job as the recommended way.

Reply:
Fine with the change.